### ISSUE TITLE
`watch` command doesn't work properly

### ISSUE THREAD

**Issue by MythreyaK** [microsoft/terminal#8869]
# Environment

```none
Windows build number: 10.0.19042.0
Windows Terminal version (if applicable): 1.5.3242.0

Any other software? 
OpenSSH_for_Windows_8.1p1, LibreSSL 2.9.2
```

# Steps to reproduce

Run `watch` command

# Expected behavior

Should look like the powershell output in the image (1) below 

# Actual behavior

Confusing text wrapping as shown in the image (WT in the background)

There's also the scrollbar that appears when using `watch`, but ideally, I don't think the scrollbar should be available. Powershell also faces this issue ONLY when there's a previous scrollback. 

### 1. Powershell and WT. Powershell has no scrollback now, so looks okay

![image](https://user-images.githubusercontent.com/26112391/105632324-79f33300-5e78-11eb-850d-1669f42fc17f.png)

### 2. Powershell and WT both have previous scrollback, so both are 'broken'

![image](https://user-images.githubusercontent.com/26112391/105632406-e8d08c00-5e78-11eb-8c54-646acdd3b09a.png)

### 3. WT when scrolled up when running watch

![wt](https://user-images.githubusercontent.com/26112391/105633092-a0b36880-5e7c-11eb-9d64-055c15830119.gif)

**Comment by zadjii-msft** [comment]
OKay I'm not _really_ sure what's going on here. Does this repro with WSL, _without_ `ssh`? Because just running `watch ls` locally _seems_ mostly fine to me. 

I think `watch` wants to be using the alt buffer, which conpty & WT don't yet support (#381). That would be what's causing the scrollbar to appear. I'm not really sure what else is wrong exactly here.

**Comment by MythreyaK** [comment]
You're right, `WSL` and `ssh` through `WSL` works fine. `CMD`, `Powershell` and `Powershell Core` all face the issue I describe above

**Comment by DHowett** [comment]
Sorry, would you mind clarifying?

You're using `watch`:

* [ ] through WSL
* [ ] through `ssh` inside WSL
* [ ] through PowerShell
* [ ] through `ssh` from PowerShell
* [ ] through CMD
* [ ] through `ssh` from CMD

?

**Comment by MythreyaK** [comment]
It seems to be triggered sometimes by running the `clear` command in an active ssh session, and resizing the window sometimes (resize without watch running). 

❌ Means there was at the least one instance where I had the problem. 
✔ means I couldn't reproduce the issue as far as I tried (clear screen, resize window b/w steps, backtracking to previous steps and trying again. 

For example, a series of steps could be `wsl` → `powershell core` → `ssh` → `watch date` → stop watching → `clear` → exit ssh → `clear` → `ssh` → `watch date` that failed)

✔ `wsl` → `watch date`
✔ `wsl` → `ssh` → `watch date`
❌ `wsl` → `powershell core`/`cmd`/`powershell` → `ssh` → `watch date`
❌ `Powershell Core` → `ssh` → run `clear` → `watch date`

**Comment by DHowett** [comment]
Okay, good! Thanks! This has the same root cause as /dup https://github.com/PowerShell/openssh-portable/pull/426. The fix should be available in versions of Win32-OpenSSH greater than 7.7 :smile:

**Comment by MythreyaK** [comment]
I found it curious that `wsl` worked, while others didn't, since from my understanding, `wsl` also uses the same shell infrastructure (`conhost`? and `conpty`) that `cmd` etc., use. I tried a few more combinations and nesting, and I think I've found the root cause. 

It seems to be an issue with `ssh` on windows, instead of the terminal. Because `ssh` from `wsl` uses the inbuilt `ssh` (at `/usr/bin/ssh` binary, it works fine, but as soon as I call `ssh` that's on windows (with `ssh.exe`), the issue appears.

**Comment by MythreyaK** [comment]
Ahh you beat me to it :)
I'm on `OpenSSH_for_Windows_8.1p1, LibreSSL 2.9.2`, so not sure what's happening

**Comment by DHowett** [comment]
It may be worth raising this issue on their tracker to figure out when it's going to be released -- I thought that 8.1 would contain recent fixes, but perhaps it does not.

**Comment by MythreyaK** [comment]
Okay, thank you very much!
